### PR TITLE
286 sidebar width

### PR DESCRIPTION
**Pull request by imaustink**
The start of this gif demonstrates a reload.
canjs-search-not-janky

### CODE REVIEW

File: static/canjs.js [hunk 1]
```diff
@@ -28,9 +28,6 @@ var $articleContainer,
 	hasShownSearch = false;
 	init();
 
-	// prevent sidebar from changing width when header hides
-	$('#left').css('min-width', $('.top-left').width());
```
Comment (chasenlehara) [inline static/canjs.js]: Is this no longer needed at all?

**Comment by imaustink** [comment]
Updated PR
search-width

File: static/search.js [hunk 2]
```diff
@@ -589,7 +591,9 @@ var Search = Control.extend({
 			});
 
 			this.$resultsContainer.scrollTop(0);
+			$('#left').addClass('search-showing');
 		}
+		$('#left').addClass('search-showing');
```
Comment (chasenlehara) [inline static/search.js]: Either this one shouldn’t be here or line 594 shouldn’t be there.

File: static/search.js [hunk 3]
```diff
@@ -565,6 +566,7 @@ var Search = Control.extend({
 				}
 			});
 		}
+		$('#left').removeClass('search-showing');
```
Comment (chasenlehara) [inline static/search.js]: Either this one shouldn’t be here or line 552 shouldn’t be there.

**Pull request merged**